functions-and-scope.js

```diff
@@ -16,6 +17,14 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+let gradeCumLaude = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+        gradeCumLaude += 1;
```

Comment:
Je zou het nóg iets korter kunnen schrijven:
```suggestion
        gradeCumLaude++;
```